db/seeds.rb.example

```diff
@@ -0,0 +1,20 @@
+# Replace <your gmail name> with your gmail name below.
+
+Community.create!([
```

Comment:
Square brackets are not needed. Neither actually are the curlies. If you give a hash as the last paramater to a ruby function, you can omit the curly braces.